plugins/woocommerce/src/Internal/Utilities/WebhookUtil.php

```diff
@@ -0,0 +1,136 @@
+<?php
+/**
+ * WebhookUtil class file.
+ */
+
+namespace Automattic\WooCommerce\Internal\Utilities;
+
+use Automattic\WooCommerce\Internal\Traits\AccessiblePrivateMethods;
+
+/**
+ * Class with utility methods for dealing with webhooks.
+ */
+class WebhookUtil {
+
+	use AccessiblePrivateMethods;
```

Comment:
Side note not affecting the acceptance: Do you think there should be some convention where we briefly document why this is being used and add a reference to our docs or sth like that? I'm just trying to make sure ppl understand the implications etc.